src/components/TalentsListPage/SearchPanel/SearchPanel.jsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from "react";
+import styles from "./SearchPanel.module.scss";
+//import st from "../../Layout/Header/Header.module.scss";
+export function SearchPanel() {
+    const inputNames = useMemo(
+        () => [
+            { name: "Job title" },
+            { name: "Location" },
+            { name: "Distance" },
+        ],
+        []
+    );
+    return (
+        <div className={styles["search-panel"]}>
+            {inputNames.map(({ name }) => (
```

Comment:
you shouldnt use map() for this inputs here, these inputs arent iterable objects, they must be kept here on a permanent basis and their number must not change